### PR TITLE
listing fee tooltip

### PR DESCRIPTION
**Pull request by PiranavanShanmugavadivelu**

### CODE REVIEW

**Review by ba1ciu — APPROVED** [review]

**Comment by gjermundgaraba** [comment]
Not related to the PR, but so good that we can make these changes so quickly (from idea to done, even if it is pretty small change) 🚀

**Review by jschill — CHANGES_REQUESTED** [review]
Awesome! But i think we should create a Tooltip-component so we don't accidently use different looks on the tooltips

File: frontend/marketplace/src/components/ui/Tooltip.vue [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<script setup lang="ts">
+interface Props {
+  label: string;
+  iconStyle: string;
+}
+const props = defineProps<Props>();
+</script>
+<template>
+  <p class="tooltip" :data-tip="props.label" data-tip-class="my-custom-tooltip">
```
Comment (jschill) [inline frontend/marketplace/src/components/ui/Tooltip.vue:9]: Remove data-tip-class?

File: frontend/marketplace/src/components/ui/Tooltip.vue [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<script setup lang="ts">
+interface Props {
+  label: string;
+  iconStyle: string;
```
Comment (jschill) [inline frontend/marketplace/src/components/ui/Tooltip.vue:4]: Maybe `iconClass` so we know its the class and not the style attribute that we're talking about

File: frontend/marketplace/src/components/ui/Tooltip.vue [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+<template>
+  <p class="tooltip" :data-tip="props.label" data-tip-class="my-custom-tooltip">
+    <img :class="props.iconStyle" src="../../assets/questionIcon.svg" />
+  </p>
+</template>
+<style>
+.tooltip {
+  --tooltip-color: #000000;
+  --tooltip-text-color: #ffffff;
+  --tooltip-tail: 10px;
```
Comment (jschill) [inline frontend/marketplace/src/components/ui/Tooltip.vue:17]: Niiiiiice

**Review by jschill — CHANGES_REQUESTED** [review]

**Review by jschill — APPROVED** [review]

**Pull request merged**